### PR TITLE
fix(test): Update e2e assert descriptions

### PR DESCRIPTION
**Pull request by djansen-redhat**
JIRA issue URL: n/a

Include .as descriptions in end to end tests. Also fixes some missing assertions.

## Checklist

- JIRA link
- Check target branch
- Make sure all commit statuses are green or otherwise documented reasons to ignore
- QA needs to evaluate against the JIRA ticket
- Changed files and commits make sense for this PR

See [Zanata Development Guidelines](https://github.com/zanata/zanata-platform/wiki/Development-Guidelines) more for information.

----
*This template can be updated in .github/PULL_REQUEST_TEMPLATE.md*

### CODE REVIEW

**Comment by kathryngo** [comment]
:lgtm:

---

Reviewed 2 of 2 files at r1.
Review status: :shipit: complete! all files reviewed, all discussions resolved

---



*Comments from [Reviewable](https://reviewable.io/reviews/zanata/zanata-platform/903#-:-LEvlJGDESxQx0z3amVr:bnfp4nl)*

File: server/functional-test/src/test/java/org/zanata/feature/endtoend/UserEndToEndTest.java [hunk 1]
```diff
@@ -377,7 +402,8 @@ private VersionLanguagesPage downloadTranslatedDocuments(EditorPage editorPage)
                 .clickVersionBreadcrumb("master")
                 .clickLocale("en-US")
                 .clickDownloadTranslatedFile(testFile.getName(), "txt");
-        assertThat(new File("/tmp/" + testFile.getName()).exists());
+        assertThat(new File("/tmp/" + testFile.getName()).exists())
+                .as("The file was successfully downloaded");
```
Comment (seanf) [inline server/functional-test/src/test/java/org/zanata/feature/endtoend/UserEndToEndTest.java]: https://thumbs.gfycat.com/WeeEagerAlaskankleekai-size_restricted.gif

**Comment by seanf** [comment]
Reviewed 1 of 1 files at r2.
Review status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @djansen-redhat)

---

*[server/functional-test/src/test/java/org/zanata/feature/endtoend/UserEndToEndTest.java, line 311 at r2](https://reviewable.io/reviews/zanata/zanata-platform/903#-LEy5caU6mHXZZtYR-cw:-LEy5caV5_4X5214HWAr:bdoe6yo) ([raw file](https://github.com/zanata/zanata-platform/blob/fe6f1d2215cd94ee79f0283fcc78f225146ec86c/server/functional-test/src/test/java/org/zanata/feature/endtoend/UserEndToEndTest.java#L311)):*
> ```Java
>                 .clickAddPerson();
>         assertThat(projectPeoplePage.getPeople().contains("translator|Translator"))
>                 .as("The user was added to the project");
> ```

Looks like another one in the wrong order. But at least the assertion is there. Not a blocker.

---


*Comments from [Reviewable](https://reviewable.io/reviews/zanata/zanata-platform/903)*

**Comment by seanf** [comment]
Reviewed 1 of 2 files at r1, 1 of 1 files at r3.
Review status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @djansen-redhat)

---



*Comments from [Reviewable](https://reviewable.io/reviews/zanata/zanata-platform/903)*

**Comment by djansen-redhat** [comment]
Reviewed 13 of 13 files at r5.
Review status: :shipit: complete! all files reviewed, all discussions resolved

---



*Comments from [Reviewable](https://reviewable.io/reviews/zanata/zanata-platform/903)*

**Comment by seanf** [comment]
Good to go, assuming full functional tests pass.

**Comment by seanf** [comment]
Reviewed 1 of 1 files at r6.
Review status: :shipit: complete! all files reviewed, all discussions resolved

---



*Comments from [Reviewable](https://reviewable.io/reviews/zanata/zanata-platform/903)*

**Pull request merged**